### PR TITLE
DOC: add examples to numpy.char.replace

### PR DESCRIPTION
**Pull request by simranmakandar**

### CODE REVIEW

File: numpy/core/defchararray.py [hunk 1]
```diff
@@ -1187,7 +1187,15 @@ def replace(a, old, new, count=None):
...
     str.replace
+    
+    Examples
+    --------
+    >>> import numpy as np 
+    >>> print(np.char.replace('That is a mango', 'mango', 'banana'))
+    That is a banana
 
+    >>> print(np.char.replace('The dish is fresh', 'is', 'was'))
+    The dwash is fresh
```
Comment (tylerjereddy) [inline numpy/core/defchararray.py:1198]: In case it helps, when I run this example locally I get:

`The dwash was fresh`

Comment (simranmakandar) [reply numpy/core/defchararray.py:1198]: yes that's correct, it must be 'The dwash was fresh'. I wanted to show the example for how 'is' in all places is replaced by 'was'. How can I change it?

Comment (tylerjereddy) [inline numpy/core/defchararray.py:1198]:
```suggestion
    The dwash was fresh
```

One approach is to suggest a change on your own pull request like above, from "Files Changed" tab, and then commit it through the interface.

**Comment by seberg** [comment]
Looks good to me. @simranmakandar if you are up for it, I think it would be cool to make the example use an array, so that they start with something like: `a = np.array(["That is a mango", "Monkeys eat mangos"])` or so and you can see that it operates on multiple elements at once.

**Review by rossbar — APPROVED** [review]
LGTM, thanks @simranmakandar . Let's put this in once the doctest checks complete.

**Comment by InessaPawson** [comment]
Hi-five on merging your first pull request to NumPy, @simranmakandar! We hope you stick around! Your choices aren’t limited to programming – you can review pull requests, help us stay on top of new and old  issues, develop educational material, work on our website, add or improve graphic design, create marketing materials, translate website content, write grant proposals, and help with other fundraising initiatives. For more info, check out: https://numpy.org/contribute
Also, consider joining our mailing list. This is a great way to connect with other cool people in our community and be part of important conversations that affect the development of NumPy: https://mail.python.org/mailman/listinfo/numpy-discussion

**Pull request merged**